### ISSUE TITLE
Integration broken with HA Core 2024.10.x

### ISSUE THREAD

**Issue by jvuotila** [custom-components/nordpool#411]
## Version of the custom_component
0.0.14

## Homeassistant version
Core 2024.10.1
Supervisor 2024.10.0
Operating System 13.1
Frontend 20241002.2

## Describe the bug
Integration use depracated HA Core helper function.

Logger: homeassistant.helpers.template
Source: helpers/deprecation.py:197
First occurred: 10:45:34 (1 occurrences)
Last logged: 10:45:34

attach was called from nordpool, this is a deprecated function which will be removed in HA Core 2025.10. Use automatic setting of Template.hass introduced by HA Core PR #89242 instead, please create a bug report at https://github.com/custom-components/nordpool/issues

**Comment by jvuotila** [comment]
Seems that Nordpool has issues. Not the integration.

**Comment by Cougar** [comment]
WDYM? This is integration code that uses deprecated call. Maybe OP didn't format it the best way. This is the log line:
```
2024-10-08 11:19:15.625 WARNING (MainThread) [homeassistant.helpers.template] attach was called from nordpool, this is a deprecated function which will be removed in HA Core 2025.10. Use automatic setting of Template.hass introduced by HA Core PR #89242 instead, please create a bug report at https://github.com/custom-components/nordpool/issues
```

**Comment by Hellowlol** [comment]
Are you using the latest version?

**Comment by Hellowlol** [comment]
I forgot to update the manifest I pretty sure that the reason for why you was not notified.

**Comment by velodromen** [comment]
Yes that is likely the cause here as I noticed all my other hacs integrations  were up to date. Never the less, it's working great and I'm a happy camper! Thanks again for quick and great support!
________________________________
Från: Steffen Fredriksen ***@***.***>
Skickat: den 18 oktober 2024 07:43
Till: custom-components/nordpool ***@***.***>
Kopia: velodromen ***@***.***>; Comment ***@***.***>
Ämne: Re: [custom-components/nordpool] Integration broken with HA Core 2024.10.x (Issue #411)


I forgot to update the manifest I pretty sure that the reason for why you was not notified.
